packages/account-tree-controller/src/AccountTreeController.test.ts

```diff
@@ -1118,6 +1153,55 @@ describe('AccountTreeController', () => {
     });
   });
 
+  describe('account ordering by type', () => {
+    it('orders accounts in group according to AccountTypeOrder regardless of insertion order', () => {
+      const evmAccount = MOCK_HD_ACCOUNT_1;
+
+      const solAccount = {
+        ...MOCK_SNAP_ACCOUNT_1,
```

Comment:
Just to make the test a bit more clear, I'd add `id: 'mock-sol-id-1'` (instead of just `mock-snap-id-1`)